<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loggingconfiguration</class>
 <widget class="QWidget" name="Loggingconfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logging Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="general_spinSetInterval">
     <property name="maximum">
      <double>3600.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="pushBrowseFileLocation">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLineEdit" name="lineFilelocation"/>
   </item>
   <item row="9" column="3">
    <widget class="QDialogButtonBox" name="buttonBox_finish">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Note: nothing will be overwritten, even if you choose a pre-existing database file!</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="general_spinSetInterval_Live">
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Logfile location: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Live Logging Interval:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Database Logging Interval:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
